csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/DependencyManager.Nuget.cs

```diff
@@ -587,19 +581,11 @@ private IEnumerable<string> GetFeeds(Func<IList<string>> getNugetFeeds)
             }
         }
 
-        private (HashSet<string>, HashSet<string>) GetAllFeeds(List<FileInfo> allFiles)
+        private (HashSet<string> explicitFeeds, HashSet<string> allFeeds) GetAllFeeds(List<FileInfo> allFiles)
         {
-            IList<string> GetNugetFeeds(string nugetConfig)
-            {
-                logger.LogInfo($"Getting Nuget feeds from '{nugetConfig}'...");
-                return dotnet.GetNugetFeeds(nugetConfig);
-            }
+            IList<string> GetNugetFeeds(string nugetConfig) => dotnet.GetNugetFeeds(nugetConfig);
```

Comment:
You can just call `dotnet.GetNugetFeeds` directly instead of declaring a local function (same comment below).